cmake/projects/cryptopp/hunter.cmake

```diff
@@ -8,6 +8,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 
 # List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    cryptopp
+    VERSION
+    "5.6.5-p0"
```

Comment:
Update default version in `cmake/configs/default.cmake`:
* https://docs.hunter.sh/en/latest/creating-new/update.html#update-package

Reply:
@ruslo: updated.